server/index-node/src/schema.graphql

```diff
@@ -22,6 +31,10 @@ type Query {
   subgraphFeatures(subgraphId: String!): SubgraphFeatures!
   entityChangesInBlock(subgraphId: String!, blockNumber: Int!): EntityChanges!
   blockData(network: String!, blockHash: Bytes!): JSONObject
+  cachedEthereumCalls(
+    network: String!
+    blockHash: Bytes!
```

Comment:
Sorry I didn't see you've addressed [my comment](https://github.com/graphprotocol/graph-node/issues/3312#issuecomment-1074149494) before hand 😅